src/course/syllabus/developer/projects/DOTNET-intro/project.md

```diff
@@ -0,0 +1,55 @@
+## Intro
+
+Learners will build a library sorting and sharing some sort of information music, photos recipes etc. Users will be able to make personal lists and view items and lists created by other users. Users can upload and favorite items.
+
+The frontend of this project will be built using React. For the backend, it will use ASP.NET to handle server-side functionality. User accounts and item lists will be stored in a PostgreSQL or SQLite database.
+
+## Project
+
+Your project for the next 3 weeks will be to develop a .NET backend connected to a database and linked to a React frontend. This week you should start by building the backend and testing it with swagger to see that you can create users with libraries containing lists of items. See if you can create a database and make calls to find lists by usernames or to find or filter lists.
+
+## Spike
+
+Like the last project you will benefit this project by building a scheme for your database before you start. Hopefully with what you learned from last project you are already ready to think about how things will connect. For this week making the backend also be sure to consider what sorts of routes you will want. Finally think about what you have done for stretch goals on previous projects and see if you can come up with a different sort of goal this time for example if you've been stretching yourself by adding technical complexity try stretching yourself by having a really nice looking site this time.
+
+### Questions to consider
+
+1. What routes will you need.
+2. What sort of objects will you need to match these data types and what will these look like in dotnet.
+
```

Comment:
.NET instead of dotnet.
Also, maybe because I don't understand .NET, but point two is a little unclear. Do you mean classes? Like something about the shape of the data you will be receiving  and how that might be represented in OOP?